src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -371,6 +410,147 @@ private static Properties readConfigFileProperties(final Path configFilePath) {
         return configProperties;
     }
 
+    /**
+     * Logs the meta data about the data sources from the given {@code dataSourcesPath}.
+     * We assume the data sources path is OK in the case that the version information cannot be read because the
+     * user can create their own data sources directory, which may not contain the metadata we seek.
+     * @param dataSourcesPath {@link Path} to a Data Sources directory to check.
+     * @return {@code True} if the given {@code dataSourcesPath} is equal to or newer than the minumum version, or if the version is unreadable.  {@code False} otherwise.
+     */
+    private static boolean logDataSourcesInfo(final Path dataSourcesPath) {
+
+        boolean dataSourcesPathIsAcceptable = true;
+
+        final Path readmePath = dataSourcesPath.resolve(IOUtils.getPath(README_FILE_NAME));
```

Comment:
Where is documentation about the format of the README file?

Reply:
I'll add in an explanation of what is required in the README for these purposes.